code/controllers/subsystem/voting.dm

```diff
@@ -116,7 +116,18 @@ var/datum/subsystem/vote/SSvote
 					else
 						master_mode = .
 	if(restart)
-		world.Reboot("Restart vote successful.", "end_error", "restart vote")
+		if(!admins.len)
+			return
+		var/total_admins
+		var/afk_admins
+		for(var/client/C in admins)

```

Comment:
what is the point of calculating total admins and afk admins then subtracting them from each other?
    
    for(var/c in admins)
        if(!c.is_afk()) 
            active_admins++

etc